eng/Versions.props

```diff
@@ -54,7 +54,7 @@
     <_MicrosoftWebWebView2Version>1.0.1774.30</_MicrosoftWebWebView2Version>
     <!-- GLIDE - the android maven artifact in /src/Core/AndroidNative/maui/build.gradle -->
     <!-- must be kept in sync with the binding library version to it here: -->
-    <_XamarinAndroidGlideVersion>4.14.2.1</_XamarinAndroidGlideVersion>
+    <_XamarinAndroidGlideVersion>4.15.1</_XamarinAndroidGlideVersion>
```

Comment:
@jonathanpeppers  this is bringing that error :

```
MSBUILD : java.exe error JAVA0000: Directory 'obj\Release\net8.0-android\lp\133' is from 'androidx.lifecycle.lifecycle-runtime-ktx.aar'. [D:\a\_work\1\s\src\Essentials\test\DeviceTests\Essentials.DeviceTests.csproj::TargetFramework=net8.0-android]
```

https://dev.azure.com/xamarin/public/_build/results?buildId=87626&view=logs&j=5239058e-6f59-5db5-6c0e-6f79eefcd049&t=5a96ad5f-9988-5704-46b8-08f3710aee54&l=13748

Reply:
@moljac @jpobst have we seen something like this before?

Reply:
@rmarinho they are saying it is: https://github.com/xamarin/AndroidX/issues/717

And the fix is to update to the latest/fixed packages.

Reply:
or try this

https://github.com/xamarin/AndroidX/issues/717#issuecomment-1571652019

Reply:
As temp workaround try explicitly adding following PackageReferences:

```
		<PackageReference Include="Xamarin.AndroidX.Activity" Version="1.7.2" />
		<PackageReference Include="Xamarin.AndroidX.Activity.Ktx" Version="1.7.2" />
```

Reply:
@rmarinho 

Try updating all AX packages at once. If you update Lifecycle package one and something else references older version[s] - duplicate classes may happen.

The same will apply for GPS-FB-MLKit after we merge and publish "alignment" pacakges.

Reply:
@rmarinho 

If you want me I could prepare PR with Android packages bump after we merge GPS-FB-MLKit

Reply:
My guess would be:
```xml
<PackageReference Include="Xamarin.AndroidX.Lifecycle.Runtime.Ktx" Version="2.6.1.1" />
```

Reply:
@moljac yeah I think we need to make a PR with all android x updates to net8

Reply:
I reverted glide to see if it fixes it

Reply:
@rmarinho 

If you want me I could prepare PR with Android packages bump after we merge GPS-FB-MLKit

Reply:
> @moljac yeah I think we need to make a PR with all android x updates to net8

@rmarinho OK. I'll do it as soon as I publish GPS-FB-MLKit alignment bumps. That should reduce this kind of problems drastically.